nautobot/extras/templates/extras/approvalworkflowstageinstance_retrieve.html

```diff
@@ -0,0 +1,24 @@
+{% extends 'generic/object_retrieve.html' %}
+{% load helpers %}
+
+{% block extra_buttons %}
+    <span title="{% if not object.is_active_stage %}Current Stage is not active yet{% elif request.user in object.users_that_already_approved %}You already approved this stage{% else %}Approve this stage{% endif %}">
```

Comment:
Presumably this should also check whether the given user is in a group that *can* approve/deny this stage?